front-end/lambda-in/src/Components/Landing Page/landing-page.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import NavBar from './Navbar/navbar.js';
+import ImageContainer from './Image Container/image-container.js'
+
+class LandingPage extends React.Component {
```

Comment:
I would suggest naming the file in CamelCase, so the file will match the Class name.
```
LandingPage.js
```